magma_access_gateway_configurator/__init__.py

```diff
@@ -13,10 +13,13 @@
 from .agw_configurator import AGWConfigurator
 
 logger = logging.getLogger(__name__)
-handler = JournalHandler()
-handler.setFormatter(logging.Formatter("Magma AGW Configurator: %(message)s"))
-logger.addHandler(handler)
 logger.setLevel(logging.INFO)
+journal_handler = JournalHandler()
+journal_handler.setFormatter(logging.Formatter("Magma AGW Configurator: %(message)s"))
+logger.addHandler(journal_handler)
+stdout_handler = logging.StreamHandler(sys.stdout)
+stdout_handler.setFormatter(logging.Formatter("Magma AGW Configurator: %(message)s"))
```

Comment:
Do we need the "Magma AGW Configurator" section? when you run `magma-access-gateway.configure` you already know you're in the context of configuring magma.